css-paint-api/Overview.bs

```diff
@@ -131,10 +131,16 @@ partial interface CSS {
 
 The {{PaintWorkletGlobalScope}} is the global execution context of the {{paintWorklet}}.
 
+The {{PaintWorkletGlobalScope}} has a devicePixelRatio property, which represents the ratio
```

Comment:
You can link to devicePixelRatio via. {{PaintWorkletGlobalScope/devicePixelRatio}}